src/core/diags/factory_diags.cpp

```diff
@@ -244,6 +244,7 @@ Error Diags::ProcessRepeat(uint8_t aArgsLength, char *aArgs[], char *aOutput, si
 
         SuccessOrExit(error = ParseLong(aArgs[1], value));
         VerifyOrExit(value <= OT_RADIO_FRAME_MAX_SIZE, error = kErrorInvalidArgs);
+        VerifyOrExit(value >= (OT_RADIO_FRAME_MIN_SIZE), error = kErrorInvalidArgs);
```

Comment:
nit: to be consistent with line above
```suggestion
        VerifyOrExit(value >= OT_RADIO_FRAME_MIN_SIZE, error = kErrorInvalidArgs);
```